### ISSUE TITLE
How might we make njump new to nostr normie friendly?

### ISSUE THREAD

**Issue by alltheseas** [fiatjaf/njump#21]
https://damus.io/note1jmn7dudakx2da5uzhj50frm0fcfuppm00hvzqrmvulm3l3gx9sksnnphkw

@karnagebitcoin

**Comment by cxplay** [comment]
If one doesn't know about npub and nevent, then one should ask Google first. njump is not a user guide or wikipedia.

**Comment by alltheseas** [comment]
> If one doesn't know about npub and nevent, then one should ask Google first. njump is not a user guide or wikipedia.

This is poor user experience.

>  njump is not a user guide or wikipedia.

It is not clear to me who njump is for, if not for new to nostr folks. Is it for nostr veterans?

**Comment by cxplay** [comment]
> This is poor user experience.

No, the experience is good enough for me. It's good for sharing things in most locations outside of Nostr, and provides a good preview and reading experience. And for those who don't know what Nostr is, it provides an entry point to learn about it.

> It is not clear to me who njump is for, if not for new to nostr folks. Is it for nostr veterans?

Anyone who doesn't know what Nostr is certainly won't be using njump. Compared to the full-featured Nostr client it provides a faster and simpler event preview service than most clients.

**Comment by alltheseas** [comment]
> I find it weird that you didn't even use a njump URL in your references to Nostr events above.

It's a least friction sharing link habit of the client I am using. 

Will's note: Njump.me/note1r7d62u2gp7z52q2jradnasyy37k7pnduqcg0xnd7l4w05yf3cegsz80n03

My note: njump.me/note1jmn7dudakx2da5uzhj50frm0fcfuppm00hvzqrmvulm3l3gx9sksnnphkw

**Comment by karnagebitcoin** [comment]
> I agree with Daniele. If njump is to act as a "gateway" to the Nostr network from the outside, it must have a clearer UI and UX design.

But it already does. There are a bunch of links from Twitter going to nostr - that's how I found it in the first place.
